README.md

```diff
@@ -144,21 +144,21 @@ asap-raw-data-{cohort,team-xxyy}
         │   └── ${cellranger_task_version}
         │       └── <cellranger output>
         ├── remove_technical_artifacts
-        │   └── ${preprocess_workflow_version}
+        │   └── ${cellbender_task_version}
         │       └── <remove_technical_artifacts output>
         └── counts_to_adata
-            └── ${preprocess_workflow_version}
+            └── ${adata_task_version}
                 └── <counts_to_adata output>
 ```
 
 ### Staging data (intermediate workflow objects and final workflow outputs for the latest run of the workflow)
 
-Following QC by researchers, the objects in the dev or uat bucket are synced into the curated data buckets, maintaining the same file structure. Curated data buckets are named `asap-curated-data-{cohort,team-xxyy}`.
+Following QC by researchers, the objects in the dev or uat bucket are synced into the curated data buckets, maintaining the same file structure. Curated data buckets are named `asap-curated-data-{cohort,team-xxyy}-{source}-{dataset}`.
 
 Data may be synced using [the `promote_staging_data` script](#promoting-staging-data).
 
 ```bash
-asap-dev-data-{cohort,team-xxyy}
+asap-dev-data-{cohort,team-xxyy}-{source}-{dataset}
```

Comment:
source is redundant given the workflow... just need to make a call on consistency vs. simplicity